src/query-builder/components/AutocompleteWrapper.tsx

```diff
@@ -32,121 +33,89 @@ type SelectValue = {
   apiId?: string;
 };
 
-type AutocompleteWrapperState = {
-  data: SelectValue[];
-  previousTextInputValue: string;
-  isLoading: boolean;
-};
+const minAPISuggesterChars = 3;
 
-const minCharsToShowDropdown = 3;
+export const prepareData = (suggestions: Suggestion[]) =>
+  suggestions.map(
+    (suggestion: Suggestion): SelectValue => ({
+      pathLabel: `${suggestion.value} [${suggestion.id}]`,
+      itemLabel: suggestion.value,
+      apiId: suggestion.id,
+      id: v1(),
```

Comment:
It looks like in Franklin it's used to set a react key. Not asking for that to be modified at this point, but I'm wondering if you know why this couldn't be `suggestion.id`? Could it be the case that 2 suggestions might have the same ID?

Reply:
According to backend we shouldn't have to worry about duplicate IDs being returned so using this for the ID now.